src/format/parsed.rs

```diff
@@ -232,13 +232,27 @@ impl Parsed {
         Parsed::default()
     }
 
-    /// Tries to set the [`year`](#structfield.year) field from given value.
+    /// Set the 'year' field to the given value.
+    ///
+    /// The value can be negative unlike the 'year divided by 100' and 'year modulo 100' fields.
```

Comment:
I think we need commas before "unlike". When referencing values stored in other fields, would suggest referencing the field directly, like `year_div_100` or even ```[`Parsed::year_div_100`]```?

Reply:
> When referencing values stored in other fields, would suggest referencing the field directly, like `year_div_100` or even `` [`Parsed::year_div_100`] ``?

I choose to avoid referencing the fields directly so it is easier to make them private (my proposal for the 0.5.x branch).

Reply:
Okay, suggest making them `#[doc(hidden)]` on this branch.

Reply:
:+1: I was think the same, I'll do it with a PR that adds the range checks and setter methods.